pages/team.md

```diff
@@ -1,38 +1,41 @@
 # Members
-## Current virtual intern lead
-* [aberdean](aberdean.md)
-
----
 ## Current virtual interns
-* [FanisGk](FanisGk.md)
-* [mepda](mepda.md)
-* [prmurphy](prmurphy.md)
-* [hikaruit15](hikaruit15.md)
-* [waryhermit](waryhermit.md)
-* [IshanSoni](IshanSoni.md)
-* [hannahgdubin](hannahgdubin.md)
+|**Username**|**Join Date**|
+|------------|-------------|
+|[aberdean](aberdean.md)(Internlead)|   |
+|[FanisGk](FanisGk.md)|   |
+|[mepda](mepda.md)|   |
+|[prmurphy](prmurphy.md)|   |
+|[hikaruit15](hikaruit15.md)|   |
+|[waryhermit](waryhermit.md)|20161213|
+|[IshanSoni](IshanSoni.md)|   |
+|[hannahgdubin](hannahgdubin.md)|   |
+
 
 if you have any questions or need help completing the steps, feel free to ask any of the current interns on [Gitter chat](https://gitter.im/open-learning-exchange/chat)
 
 ## Previous Interns
 ---
->### Summer 2016 virtual intern lead
-* [EmilyLarkin](EmilyLarkin.md)
-* [DrewPerlman](DrewPerlman.md)
-* [i5o](i5o.md)
-
->---
-### Summer 2016 virtual interns
-* [fsjoyti](fsjoyti.md)
-* [xinglunxu](xinglunxu.md)
-* [pranesh-s](pranesh-s.md)
-* [epistulae](epistulae.md)
-* [ctl74100](ctl74100.md)
-* [AdityaNair99](AdityaNair99.md)
-* [Hulmani](Hulmani.md)
-* [sthak004](sthak004.md)
-* [Neha1990](Neha1990.md)
-* [cw-lin](cw-lin.md)
-* [314159265359](314159265359.md)
```

Comment:
here you get that error from ...

you yourself added already something to the page
@fangninghe did you ever fix master on your fork again? I mean since `sync git fork` step ;)